refactor-first-maven-plugin/src/main/java/org/hjug/mavenreport/RefactorFirstJsonReport/JsonReport.java

```diff
@@ -0,0 +1,14 @@
+package org.hjug.mavenreport.RefactorFirstJsonReport;
+
+import java.util.List;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+class JsonReport
```

Comment:
I've always been partial to having opening curly braces on the same line - could you move the curly brace up?
e.g.
``` Java
```suggestion
class JsonReport {
```
```